config/credentials.yml.enc

```diff
@@ -1 +1 @@
-Zl4ZmIChG4IqsN8yFJ+Y714y+ym1Mr7PCRPy7xbu9HOdMzX1/c6VxbYTJUhtqTbLHrp2LHCkZTfz15t79lDiwuBp00VtoWB29kC3UtSYsYKebSZsxPBfL/YfG3JrtNUgdtXMzoTbatW5tvVglKDSdQpkihdH1GIaq1pnzJaeW85Jgds3i2npYW3eeeuQI4/Ppp9U3nHoW+0daxMZxfJg6OdFZ3Xa5RDDNH75zgpObKjEcXZF9VZYqyScMDwH2N3OOe4YtIRrLDRaw/tnAauFp7wM5QjpWBegsKh3eM31hV8M1arHnHlyO+EWN2NRTrmKsUKWmewrOFu9w+NiNmNEFDKzb6/NLbPDb1+3FSxIf6515SxhgktkodL5qZoNffYXm6R3uQLfGkzlM4dFs/u+c4O4ryXjC7ols+6R--XYxLKDdGWAZhwCSX--Jp7XoPCoaaLe1PtnUiPWHA==
\ No newline at end of file
+PYhAQqx1QPMgNIUD8yy2Ck9Gm2KXEI6j4T4h8D6R5yRNCelUfFZ/mTdv1VXdOT+1j8M+VXy+Adw7Ug1/aJ78VGa5Ic0GkJL+J+a8fyoKNXpRlE+RbQ66kDPCDtlpYU6f90GHfEYFqimUySAHrHIPfs5HM0qiSU5K4M5Rt7C7h0NpCjOzCpu7Cu3R6QGucLOITKotBNEcRo8DzoKF8bRj5Eg+lx3ZxSczS5ybq6GSPZxbQ/dRguJ84K/tY0USYU6DE4eIa4qqpFdlMqdQMWD5gj4YgtCeIgQrFSVnpbZ2bJlDv47xl5mKU4/IunomvohRyqSFki2GOFF9bEORkOp+SE6CVkOa8IXa6iHlyn+Zg98b8HMkj/0Da4KC8LYDxYSN1rI2aAnxpgTRK33+/lgogLYUic3Vc7ZPRaOSSFxK3SQERyR8bCEhyKVL5w6TrJIUa5f2--wnl1nth/REa83tR5--N+C+RU/Rcnqf09O/8oKETA==
```

Comment:
@jenlu33 can you revert this change since we didn't add anything to the credentials?

```
git checkout main -- config/credentials.yml.enc
```